org.eclipse.scanning.test/src/org/eclipse/scanning/test/malcolm/device/MockedMalcolmDevice.java

```diff
@@ -37,7 +37,17 @@
 import org.eclipse.scanning.api.scan.ScanningException;
 import org.eclipse.scanning.connector.epics.EpicsV4ConnectorService;
 import org.eclipse.scanning.malcolm.core.AbstractMalcolmDevice;
-
+import org.eclipse.scanning.malcolm.core.MalcolmDevice;
+import org.junit.Ignore;
+
+/**
+ * TODO: This class and its subclasses are only used in tests that are currently
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)